### ISSUE TITLE
hmmer2: 404 Error: Failed to download resource "hmmer2"

### ISSUE THREAD

**Issue by sjackman** [Homebrew/homebrew-science#3415]
```
==> Installing hmmer2 from homebrew/science
==> Downloading http://selab.janelia.org/software/hmmer/2.3.2/hmmer-2.3.2.tar.gz

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "hmmer2"
Download failed: http://selab.janelia.org/software/hmmer/2.3.2/hmmer-2.3.2.tar.gz
```

**Comment by nextgenusfs** [comment]
Eddy lab website as moved, seems like hmmer2 now resides here: http://eddylab.org/software/hmmer/2.3.2/hmmer-2.3.2.tar.gz

**Comment by sjackman** [comment]
Duplicate of #3421

**Comment by iMichka** [comment]
Has been fixed with #3430. Closing.
